backend/src/main/java/com/sep6/backend/models/Review.java

```diff
@@ -0,0 +1,32 @@
+package com.sep6.backend.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import lombok.Data;
+import org.hibernate.validator.constraints.Range;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Data
```

Comment:
@AllArgsConstructor
@NoArgsConstructor
I thin you need those as well, maybe the builder if you have use case for it